src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/PdlClient.kt

```diff
@@ -95,14 +99,28 @@ class PdlClient(val pdlConfig: PdlConfig,
      */
     @Cacheable("personidenter", cacheManager = "shortCache")
     fun hentPersonidenter(ident: String, historikk: Boolean = false): PdlIdenter {
-        val pdlPersonRequest = PdlIdentRequest(variables = PdlIdentRequestVariables(ident, "FOLKEREGISTERIDENT", historikk),
-                                               query = PdlConfig.hentIdentQuery)
+        val pdlIdentRequest = PdlIdentRequest(variables = PdlIdentRequestVariables(ident, "FOLKEREGISTERIDENT", historikk),
+                                              query = PdlConfig.hentIdentQuery)
         val pdlResponse: PdlResponse<PdlHentIdenter> = postForEntity(pdlConfig.pdlUri,
-                                                                     pdlPersonRequest,
+                                                                     pdlIdentRequest,
                                                                      httpHeaders())
         return feilsjekkOgReturnerData(ident, pdlResponse) { it.hentIdenter }
     }
 
+    /**
+     * @param identer Identene til personene, samme hvilke type (Folkeregisterident, aktørid eller npid)
+     * @return map med søkeident som nøkkel og liste av folkeregisteridenter
+     */
+    fun hentIdenterBolk(identer: List<String>): Map<String, PdlIdent> {
```

Comment:
Burde denne eller sevicen ha håndtering for att man maks kan sende med 100 identer til PDL?